02-temperature_converter/Temperature.java

```diff
@@ -0,0 +1,21 @@
+import java.util.Scanner;
+public class Temperature{
+  public static void main(String[] args) {
+    Scanner in = new Scanner(System.in);
+    double temperature = 0;
+
+
+    System.out.print("What is the temperature in degrees Celsius: ");
+    temperature += in.nextInt();
```

Comment:
you could just say `temperature = in.nextDouble();` no need to add (but yours isn't wrong)